examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -21,11 +21,12 @@
      "name": "stdout",
```

Comment:
<p>I would not call them outliers, those are simply jumps from regions of low probability to regions of much higher probability, they are bound to happen at some point and not completely unexpected.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>